constance/settings.py

```diff
@@ -10,6 +10,8 @@
 
 CONFIG_FIELDSETS = getattr(settings, 'CONSTANCE_CONFIG_FIELDSETS', {})
 
+CONFIG_COLLPASED_FIELDSETS = getattr(settings, 'CONSTANCE_COLLPASED_FIELDSETS', tuple())
```

Comment:
It seems to me that there is enough literal form.
```suggestion
CONFIG_COLLPASED_FIELDSETS = getattr(settings, 'CONSTANCE_COLLPASED_FIELDSETS', ())
```